.github/workflows/ci.yaml

```diff
@@ -0,0 +1,16 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+name: ci
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
checkout@2 :)

Reply:
Oh, this snippet was copy-pasta from nodejs-pubsub. Is that out of date already too?

Reply:
Also, I did some recent work here to actually remove the CI tests from this repo, because it doesn't have tests anymore. It's only linting at this point. (Maybe @fhinkel could comment further on that change.)

Reply:
Actually, there is indeed no `test` action in `package.json` anymore, but the README calls for running tests. So that may also need to be updated.

Reply:
this should be ground truth:

https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/node_library/.github/workflows/ci.yaml

Is there a chance `synth.py` in pubsub is ignoring that file?

Reply:
@bcoe This synth template is the same as nodejs-pubsub. The thing at question is that the `lint` section I pasted into this PR is `@v1` like nodejs-pubsub, and I think @JustinBeckwith is saying it should be `@v2`?

Reply:
👍 we should make the update in `autosynth` too, and once we do so just make sure it matches this one?